doc/source/whatsnew/v0.25.0.rst

```diff
@@ -825,6 +825,7 @@ Other API changes
 - :meth:`ExtensionArray.argsort` places NA values at the end of the sorted array. (:issue:`21801`)
 - :meth:`DataFrame.to_hdf` and :meth:`Series.to_hdf` will now raise a ``NotImplementedError`` when saving a :class:`MultiIndex` with extention data types for a ``fixed`` format. (:issue:`7775`)
 - Passing duplicate ``names`` in :meth:`read_csv` will now raise a ``ValueError`` (:issue:`17346`)
+- :meth:`infer_type` will now return integer-na for integer and np.nan mix (:issue:`27283`)
```

Comment:
can you put integer-na in quotes here; move to 1.0

Reply:
Updated and moved